src/components/utdragFraSykefravaeret/Sykmeldinger.tsx

```diff
@@ -0,0 +1,182 @@
+    : "Sykmelding uten arbeidsgiver";
+  return (
+    <ExpansionCard aria-label={title} onToggle={logAccordionOpened}>
+      <StyledExpantionCardHeader className="w-full">
+        <ExpansionCard.Title
+          as="div"
+          className="flex justify-between m-0 text-base"
+        >
+          <SykmeldingTittelbeskrivelse sykmelding={sykmelding} />
+        </ExpansionCard.Title>
+      </StyledExpantionCardHeader>
+      <ExpansionCard.Content>
+        <SykmeldingUtdragFraSykefravaretVisning sykmelding={sykmelding} />
+      </ExpansionCard.Content>
+    </ExpansionCard>
+  );
+};
+
+interface UtvidbarTittelProps {
+  sykmelding: SykmeldingOldFormat;
+}
+
+const SykmeldingTittelbeskrivelse = ({ sykmelding }: UtvidbarTittelProps) => {
+  const sykmeldingPerioderSortertEtterDato =
+    sykmeldingperioderSortertEldstTilNyest(
+      sykmelding.mulighetForArbeid.perioder
+    );
+
+  const periode = `${tilLesbarPeriodeMedArstall(
+    tidligsteFom(sykmelding.mulighetForArbeid.perioder),
+    senesteTom(sykmelding.mulighetForArbeid.perioder)
+  )}: `;
+  const graderinger = stringMedAlleGraderingerFraSykmeldingPerioder(
+    sykmeldingPerioderSortertEtterDato
+  );
+  const diagnose = `${sykmelding.diagnose.hoveddiagnose?.diagnosekode} (${sykmelding.diagnose.hoveddiagnose?.diagnose})`;
+  const erViktigInformasjon = erEkstraInformasjonISykmeldingen(sykmelding);
+  const sykmelder = sykmelding.bekreftelse.sykmelder;
+  const arbeidsgiver = arbeidsgivernavnEllerArbeidssituasjon(sykmelding);
+  const erIkkeTattIBruk = sykmelding.status === SykmeldingStatus.NY;
+  const erUtenArbeidsgiver = erSykmeldingUtenArbeidsgiver(sykmelding);
+
+  return (
+    <div className="w-full flex flex-col">
+      <div className="flex justify-between mb-2">
+        <div>
+          {periode}
+          {graderinger}
+        </div>
+        <div className="flex gap-4">
+          {erIkkeTattIBruk && (
+            <Tag variant="warning" size="small">
+              {texts.ny}
+            </Tag>
+          )}
+          {erUtenArbeidsgiver && (
+            <Tag variant="info" size="small">
+              {texts.utenArbeidsgiver}
+            </Tag>
+          )}
+          {erViktigInformasjon && (
+            <img
+              height={18}
+              alt="Viktig informasjon"
+              src={MerInformasjonImage}
+            />
+          )}
+        </div>
+      </div>
+      {sykmelding.diagnose.hoveddiagnose && (
+        <Info label={"Diagnose: "} text={diagnose} />
+      )}
+      {sykmelder && <Info label={"Sykmelder: "} text={sykmelder} />}
+      {arbeidsgiver && <Info label={"Arbeidsgiver: "} text={arbeidsgiver} />}
+      {sykmelding.yrkesbetegnelse && (
+        <Info
+          label={"Stilling fra sykmelding: "}
+          text={sykmelding.yrkesbetegnelse}
+        />
+      )}
+      {sykmelding.papirsykmelding && <PapirsykmeldingTag />}
+    </div>
+  );
+};
+
+export default function Sykmeldinger() {
```

Comment:
Vi har egentlig ikke landet veldig på noen konvensjoner rundt hvordan vi strukturerer koden vår på frontend. Har prøvd å sett litt på airbnb sine retningslinjer og synes de har mye bra. For eksempel at man ikke skal ha flere komponenter i samme fil med mindre de er pure og stateless.
Ikke noe showstopper, men her kunne man skilt ut komponenter i egne filer for å gjøre denne litt mer oversiktlig 🤔 Kan godt se på det sammen. Jeg er litt usikker selv på hva som blir riktig her. 

Ikke noe som stopper å deploye det uansett 😊